web/.cloudbuild/scripts/go.mod

```diff
@@ -2,12 +2,14 @@ module github.com/gravitational/webapps/.cloudbuild/scripts
 
 go 1.17
 
+require github.com/gravitational/trace v1.1.15
```

Comment:
This style change occurred on master.  It's not important in this PR but we may want to return it to the require block below in master.

Reply:
Could you run `go mod tidy` and commit the changes, if any? (If there are no changes then this is fine.)

Reply:
No change from my system